packages/calling-stateful-client/src/CallClientState.ts

```diff
@@ -133,6 +135,19 @@ export interface TranscriptionCallFeatureState {
   isTranscriptionActive: boolean;
 }
 
+/* @conditional-compile-remove(capabilities) */
+/**
+ * State only version of {@link @azure/communication-calling#CapabilitiesFeature}
+ *
+ * @beta
+ */
+export interface CapabilitiesFeatureState {
+  /**
+   * Proxy of {@link @azure/communication-calling#CapabilitiesFeature.capabilities}.
+   */
+  capabilities: ParticipantCapabilities;
```

Comment:
Do we want to call this `localParticipantCapabilities`?

Reply:
I know, something just seems a little funky that the type has participant in the name but not the definition itself. was just thinking readability sake, not blocking 😊